src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/IndexStatus.kt

```diff
@@ -23,7 +23,10 @@ class IndexStatus(
   var endIndexTime: LocalDateTime?,
 
   @Field(type = FieldType.Boolean)
-  var inProgress: Boolean
+  var inProgress: Boolean,
+
+  @Field(type = FieldType.Boolean)
+  var inError: Boolean = false
```

Comment:
I wonder if you need this to be Boolean? since the existing indexes won't have a value. Can't remember if giving a default value will work. What do you think @mikehalmamoj